matchms/importing/__init__.py

```diff
@@ -11,7 +11,9 @@
 For more extensive import options we recommend building custom importers using `pyteomics
 <https://github.com/levitsky/pyteomics>`_ or `pymzml <https://github.com/pymzml/pymzML>`_.
 """
-from .load_adducts import load_adducts
+from .load_adducts import load_adducts_dict
```

Comment:
One line 2 it says `Functions for importing mass spectral data`. 
Which does not cover the exported functions. Can the docstring be extended to include adducts?

Reply:
Thanks for the great review Stefan.
I am now working on it and have a few questions.
Now, I had a csv, a yaml, and a json file which indeed is a bit much. I would merge the adducts from the yaml into the csv file. The reason to chose `.csv` here for me is that it can be nicely displayed and edited as a table.
I will import this via the csv package and not via pandas.

The json file contains a dictionary for conversions. I could of course also change that into a .csv file. Do you think that's better?

To make things a bit simpler (for me...), I will also remove the filename attribute. If people want to change the table than they can do so via an issue/PR or clone the repo.

Reply:
I added that importing adducts also is part of importing.